static/js/publisher/release/components/revisionsList.js

```diff
@@ -40,22 +40,26 @@ class RevisionsList extends Component {
   }
 
   renderRow(
+    index,
     revision,
     isSelectable,
     showChannels,
     isPending,
     isActive,
-    showBuildRequest
+    showBuildRequest,
+    activeChannel
   ) {
     return (
       <RevisionsListRow
         key={`revision-row-${revision.revision}`}
+        index={index}
         revision={revision}
         isSelectable={isSelectable}
         showChannels={showChannels}
         isPending={isPending}
         isActive={isActive}
         showBuildRequest={showBuildRequest}
+        channel={activeChannel}
```

Comment:
Is `channel` used by `RevisionsListRow` component? I don't see it in the props there